package.json

```
@@ -46,8 +46,8 @@
     "lint:package-jsons": "tsx ./scripts/lint-package-jsons",
     "type:check": "turbo run type:check",
     "type:check-tests": "tsc -p tsconfig.test.json",
-    "prettier:check": "prettier --check packages --check apps/docs-snippets",
-    "prettier:format": "prettier --write packages --check apps/docs-snippets",
+    "prettier:check": "prettier --check packages --check apps/docs",
+    "prettier:format": "prettier --write packages --write apps/docs",
```

Is this `--write`  intentional? 

```suggestion
    "prettier:format": "prettier --write packages --check apps/docs",
```

---

Yes, I'd expect to fix the changes, as it's the `prettier:format` command.